quiz.css

```diff
@@ -0,0 +1,59 @@
+* {
+	font-family: sans-serif;
```

Comment:
You are still using tabs in your css file while using spaces in your HTML. Keep them consistent.